patches/@nrk+core-datepicker+3.0.8.patch

```diff
@@ -114,11 +114,16 @@ index 00848d7..1bc38dc 100644
  
    var dashCase = name.replace(/.[A-Z]/g, function (ref) {
      var a = ref[0];
-@@ -747,6 +805,7 @@ function customElementToReact(elementClass, options) {
+@@ -746,10 +804,9 @@ function customElementToReact(elementClass, options) {
+   );
  }
  
- var coreDatepicker = customElementToReact(CoreDatepicker, {
+-var coreDatepicker = customElementToReact(CoreDatepicker, {
++export default customElementToReact(CoreDatepicker, {
```

Comment:
`rollup-plugin-commonjs` klarte ikke å dekode eksporten herfra, selv med `namedExports`, så måtte lage en liten frankenmodule 🤷‍♂️

Reply:
Hmm... Dette vil virke fordi vi transpilerer denne koden, det som er ideen? For denne syntaxen skjønner ikke IE11.

Reply:
Nettopp. Blir transpilert tilbake til cjs, men Rollup måtte ha hjelp til å importere det først.